kubernetes/loculus/templates/keycloak-deployment.yaml

```diff
@@ -18,6 +18,8 @@ spec:
       labels:
         app: loculus
         component: keycloak
+      annotations:
```

Comment:
Will this then also cause keycloak to restart when there is a change?

We haven't restarted keycloak in a while - and I recall Cornelius and I noticed that a keycloak restart means temporarily logging all users out - so maybe we don't want to add this for keycloak and really only restart when the pod needs to restart.